glitter_events/templatetags/events_tags.py

```diff
@@ -7,11 +9,15 @@
 
 
 @register.assignment_tag
-def get_upcoming_events(count=5, category=None):
+def get_upcoming_events(count=5, category='', tags=''):
     event_list = Event.objects.published().filter(start__gt=timezone.now())
 
     # Optional filter by category
-    if category is not None:
+    if category.strip() is not '':
```

Comment:
You mean `if category.strip()`?

If there's confusion here, then the verbosity of the full if statement doesn't bother me.